packages/verified-fetch/src/index.ts

```diff
@@ -784,34 +792,75 @@ export interface VerifiedFetchInit extends RequestInit, ProgressOptions<BubbledP
   allowInsecure?: boolean
 }
 
+type Libp2pServiceType = Pick<DefaultLibp2pServices, 'dcutr' | 'identify' | 'keychain' | 'ping' > & Record<`delegatedRouting${number}`, any>
+
 /**
  * Create and return a Helia node
  */
 export async function createVerifiedFetch (init?: Helia | CreateVerifiedFetchInit, options?: CreateVerifiedFetchOptions): Promise<VerifiedFetch> {
+  let libp2p: Libp2p<Libp2pServiceType> | undefined
   if (!isHelia(init)) {
-    init = await createHeliaHTTP({
+    const libp2pConfig = libp2pDefaults()
+
+    libp2pConfig.start = false
+
+    libp2pConfig.transports = [webSockets(), webRTCDirect()]
+
+    libp2pConfig.peerDiscovery = [] // disable default bootstrap peers
+    libp2pConfig.addresses = {} // disable default listen addresses
+
+    // We only need client/listen/fetch based services
+    const fetchOnlyServices = {
+      dcutr: libp2pConfig.services.dcutr,
+      dht: kadDHT({
```

Comment:
My intention was to disabling the DHT client. So this can be removed. 

Unless there's a strong reason to include, I think relying on delegated routing is an ok compromise.